.github/workflows/formatting.yml

```diff
@@ -56,8 +56,7 @@ jobs:
       with:
         environment-file: env.yml
         environment-name: deepchem
-        channels: conda-forge,defaults
```

Comment:
Can you point me to the place conda-forge is added now in the new setup workflow?

Reply:
they have been moved to the env.yml files, we don't have to explicitly mention them again in the workflow files again